src/app/(auth)/forgot-password/page.tsx

```diff
@@ -31,7 +31,7 @@ export default function ForgotPassword() {
     <>
       {!emailSentCount && (
         <>
-          <H1 $color={COLORS.blueDarker}>Forgot Password?</H1>
+          <AuthHeading>Forgot Password?</AuthHeading>
```

Comment:
Not sure if you're done yet with styling since this is a draft PR, but it looks like the text styles aren't being applied here? 

**Screenshot from your branch:**
<img width="690" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/2d90f942-025b-4f98-b780-f3dc599db1a4">

**Screenshot from Figma:**
<img width="723" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/59b48746-06ef-4d60-a3f6-65a1ea37f842">

Reply:
text weights should be updated now, also i center aligned instead of left aligning to match the Figma!